internal/olm/client.go

```diff
@@ -64,7 +64,8 @@ func ClientForConfig(cfg *rest.Config) (*Client, error) {
 	return c, nil
 }
 
-func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourceclient.Status, error) {
+func (c Client) InstallVersion(ctx context.Context, namespace, version string) (*olmresourceclient.Status, error) {
```

Comment:
```suggestion
func (c Client) InstallVersion(ctx context.Context, olmNamespace, version string) (*olmresourceclient.Status, error) {
```
Let's claifies we have watch-namespace, operator--namespace and then now olm-namespace